src/java.desktop/share/classes/sun/awt/FontConfiguration.java

```diff
@@ -156,19 +154,12 @@ public boolean fontFilesArePresent() {
         short fontNameID = compFontNameIDs[0][0][0];
         short fileNameID = getComponentFileID(fontNameID);
         final String fileName = mapFileName(getComponentFileName(fileNameID));
-        @SuppressWarnings("removal")
-        Boolean exists = java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction<Boolean>() {
-                 public Boolean run() {
-                     try {
-                         File f = new File(fileName);
-                         return Boolean.valueOf(f.exists());
-                     }
-                     catch (Exception e) {
-                         return Boolean.FALSE;
-                     }
-                 }
-                });
+        Boolean exists = false;
+        try {
+            File f = new File(fileName);
+            exists = Boolean.valueOf(f.exists());
+        } catch (Exception e) {
+        }
         return exists.booleanValue();
```

Comment:
Looks good since now exists boolean value is returned and doesn't require  return Boolean.FALSE; in catch block.